documentation/modules/proc_configuring-ova-web-upload.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="proc_configuring-ova-web-upload_{context}"]
+
+= Configuring OVA file upload by web browser
+
+[role="_abstract"]
+You can configure Open Virtual Appliance (OVA) file upload by web browser to upload an OVA file directly to an NFS share. To configure OVA file upload, you first enable OVA appliance management in the `ForkliftController` custom resource (CR) and then enable OVA upload for each OVA provider. When you enable OVA upload for an OVA provider, the *Upload local OVA files* option populates on the provider’s *Details* page in the MTV UI.
+
+.Prerequisites
+
+* You have an NFS share to point the OVA provider at.
+* You have enough storage space in your NFS share.
+* You have a valid .ova file to upload.
+* Your .ova file has a unique file name.
+
+.Procedure
+. In the Red Hat OpenShift web console, click *Operators* > *Installed Operators*.
+. Click *Migration Toolkit for Virtualization Operator*.
++
+The Operator Details page opens in the *Details* tab.
+. Click the *ForkliftController* tab, and open the `forklift-controller` resource.
+. Click the `forklift-controller` *YAML* tab, and add the `feature_ova_appliance_management` field to the `spec` section of the `forklift-controller` custom resource (CR):
++
+Example:
++
+----
+spec:
+  ...
+  feature_ova_appliance_management: 'true'
+----
+
+. In the Red Hat OpenShift web console, click *Migration for Virtualization* > *Providers*.
+. Click the provider to open the *Details* page. Click the provider’s *YAML* tab. 
++
+For information about creating a provider, see xref:adding-source-provider_ova[Adding an Open Virtual Appliance (OVA) source provider].
+. Scroll down the provider’s YAML file to `spec.settings.applianceManagement`, and set `applianceManagement` to `'true'`:
```

Comment:
This `settings` field is not displayed by default, they will need to add it. The example yaml is good, but the explanation in this bullet point may be misleading.